### PR TITLE
doc restructure

### PR DESCRIPTION
**Pull request by eschultink**
### Features
 - group docs, rules, and api responses examples under `docs/` by source

### Change implications

 - dependencies added/changed? **no**
 - something to note in `CHANGELOG.md`? **no**

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1205490673201231

### CODE REVIEW

File: java/core/src/test/java/co/worklytics/psoxy/rules/asana/AsanaTests.java [hunk 1]
```diff
@@ -36,7 +32,7 @@ void sha() {
 
     @Test
     void workspaces() {
-        //String jsonString = asJson(exampleDirectoryPath, "users.json");
+        //String jsonString = asJson("users.json");
```
Comment (jlorper) [inline java/core/src/test/java/co/worklytics/psoxy/rules/asana/AsanaTests.java:35]: remove?

File: java/core/src/test/java/co/worklytics/psoxy/rules/msft/DirectoryTests.java [hunk 2]
```diff
@@ -120,7 +121,7 @@ void groups() {
     void groupMembers() {
         String endpoint = "https://graph.microsoft.com/v1.0/groups/02bd9fd6-8f93-4758-87c3-1fb73740a315/members?$count=true";
 
-        String jsonString = asJson(exampleDirectoryPath, "group-members.json");
+        String jsonString = asJson("sources/microsoft-365/directory/example-api-responses/original/", "group-members.json");
```
Comment (jlorper) [inline java/core/src/test/java/co/worklytics/psoxy/rules/msft/DirectoryTests.java:124]:
```suggestion
        String jsonString = asJson(DIRECTORY_API_EXAMPLES_PATH, "group-members.json");
```
use the constant too

**Review by jlorper — APPROVED** [review]
A couple of things. Good restructure, I think it is way clearer now! :+1:

**Review by aperez-worklytics — APPROVED** [review]
Cool, just the constant mentioned by @jlorper and ok for me!

**Pull request merged**